CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Update `clang-format` to 6.0 for automatic code formatting (#143)
 - Improve performance of the TSP heuristic by ~35% on all TSPLIB instances (#142)
+- Improve performance of the CVRP heuristic by ~55% on all CVRPLIB instances (#146)
```

Comment:
I think the term "perfomance" for the solving approach is expected to relate to the solution quality. What about using "speed-up" instead? (Same applies to the entry for your latest PR on the TSP heuristic)

Also you might want to replace "heuristic" with "approach" for the CVRP entry as your fix is useful throughout the whole solving phase (heuristic + local search).

Reply:
I replaced "heuristic" with "solving", since I don't think that "approach" fits very well.